Compilers.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 14
-VisualStudioVersion = 14.0.25123.0
```

Comment:
I think it has to do with using update 1 or update 2.  Essentially the VS version changes so it writes out a new number.